v20.2/manage-a-backup-schedule.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Manage a Backup Schedule
+summary: Learn about how to create, manage, and restore from a schedule of periodic backups.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> You can create schedules in CockroachDB for periodic backups. Once a [backup schedule is created](#create-a-new-backup-schedule), you can do the following:
+
+- [Set up monitoring for the backup schedule](#set-up-monitoring-for-the-backup-schedule)
+- [View and control the backup schedule](#view-and-control-the-backup-schedule)
+- [View and control a backup initiated by a schedule](#view-and-control-a-backup-initiated-by-a-schedule)
+
+## Create a new backup schedule
+
+{% include copy-clipboard.html %}
+~~~
+> CREATE SCHEDULE schedule_name
+  FOR BACKUP INTO 's3://test/backups/test_schedule_1?AWS_ACCESS_KEY_ID=123&AWS_SECRET_ACCESS_KEY=123'
```

Comment:
should we list s3 explicitly, or make it obvious that it can be any supported URI (including nodelocal, and userfile://)?